ui-spacetimechart/src/stories/draw-rectangle.stories.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { Button } from '@osrd-project/ui-core';
+import type { Meta } from '@storybook/react';
+import { clamp } from 'lodash';
+
+import '@osrd-project/ui-core/dist/theme.css';
+
+import { OPERATIONAL_POINTS, PATHS } from './lib/paths';
+import { PathLayer } from '../components/PathLayer';
+import { SimpleRect } from '../components/SimpleRect';
```

Comment:
This story imports some low-level drawing helpers to render the zoom rectangle. What's the long term plan once the feature is complete: do we expect users to implement the actual zoom functionality themselves, or will we expose helpers to handle zoom rectangle interactions?

Reply:
Hm, okay, I suppose we can keep things as-is for now and decide later!